### ISSUE TITLE
[AV_ISSUES] Monitoring Issue: Tauri might randomly have AV issues

### ISSUE THREAD

**Issue by TheRedDaemon** [UnofficialCrusaderPatch/UCP3-GUI#201]
For example, a user could not update the GUI via the updater, but by using the installer.
[The issue is known](https://github.com/tauri-apps/tauri/issues/2486), but there is not really something we can do. Even the very expensive offical signing does not prevent this is some cases.

So this should be checked sometimes for updates.

**Comment by gynt** [comment]
Does this work if our binary keeps changing? https://www.microsoft.com/en-us/wdsi/filesubmission

**Comment by TheRedDaemon** [comment]
I do not think we should do anything for now. The file submissions likely only apply to one version (maybe) and would likely only help against the Windows warnings. The updater problem seems to be in relation to AVs and not due to the OS.
